blog/robocon-2025-kickoff.md

```diff
@@ -0,0 +1,18 @@
+---
+title: A Fiery RoboCon 2025 Kick-Off
+image: /images/2025/Kickoff2025.jpeg
+date: 2024-11-02T16:00:00.000Z
+excerpt: Today we launched RoboCon 2025 with our fiery new competition!
+layout: PostLayout
+---
+Today we welcomed 15 teams from 10 schools at our kickoff event for RoboCon 2025.
+
+The day started with a presentation from three of our students about this year's competition 'Dragon's Lair'. Read the full backstory about the competition on page 2 of the [rulebook](/rulebook-2025.pdf)!
+
+In this game, the arena is split into 4 terretories, each owned by a dragon (a RoboCon robot) with it's own lair at the back of the territory (a plinth raised 15 cm above the arena floor). Each dragon has its' own Gem but this gem is in another dragon's territory, and it needs to get its gems back to its' lair. The dragon’s islands are sparse in food, with sheep being their only food source. You will score points for any cube-shaped sheep (the dragon’s food) you bring back to your territory (or to your lair for extra points).
```

Comment:
Typos: 
terretories -> territories
it's -> its (yes this is weird but i think it should be like this)
its' -> its

Reply:
Cheers, lol wish vscode markdown had a spell check